lib/services/runtime/utils.ts

```diff
@@ -48,7 +48,7 @@ const convertDeprecatedReprompt = <B extends RepromptNode>(node: B) => ({
   ...node,
   noReply: {
     ...node.noReply,
-    prompts: node.noReply?.prompts || node.reprompt ? [node.reprompt] : [],
```

Comment:
yeah pretty much goofed the logic here, this always results in `[undefined]` if `node.noReply?.prompts` is true

Reply:
oof, this looks like it was hard to catch